tests/h/admin/views/features_test.py

```diff
@@ -75,21 +77,58 @@ def test_cohorts_index_when_no_cohorts():
     assert result["results"] == []
 
 
-def test_new_cohort_creates_cohort():
+def test_new_cohort_creates_cohort_with_no_members():
     req = DummyRequest(db=db.Session)
     req.params['add'] = 'cohort'
     result = views.cohorts_add(req)
     assert isinstance(result, httpexceptions.HTTPSeeOther)
 
     result = views.cohorts_index({}, req)
     assert len(result["results"]) == 1
-    assert result["results"][0].name == "cohort"
+
+    cohort = result["results"][0]
+    assert cohort.name == "cohort"
+    assert len(cohort.members) == 0
+
+
+def create_cohort(req):
```

Comment:
I wouldn't recommend using other view functions to prepare the database. Just create a `FeatureCohort` directly and add it to the database.

Doing it this way leaves these tests (for the add and remove user from cohort views) liable to break if a different view (create a cohort) changes, which I'd say would be a surprising for somebody working on the latter view.